### PR TITLE
Adds interaction status indicator and wallet.discovery config input

### PR DESCRIPTION
**Pull request by JeffreyDoyle**
- Adds interaction status
- Adds wallet.discovery config input

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -44,6 +54,21 @@ export default function Home() {
           </button>
         ))}
       </div>
+      <div style={{ marginTop: "12px" }}>
+        <label for="manual-wallet">
```
Comment (gregsantos) [inline pages/index.js]: `htmlFor` to avoid warning

File: pages/index.js [hunk 2]
```diff
@@ -44,6 +54,21 @@ export default function Home() {
...
+          onClick={async () =>
+            await fcl
+              .config()
+              .put("discovery.wallet", discoveryWalletInputRef?.current?.value)
+          }
+        >
+          Set
+        </button>
+      </div>
       <pre>{JSON.stringify({ currentUser, config }, null, 2)}</pre>
```
Comment (gregsantos) [inline pages/index.js:108]: This seems to be blowing up after authn with dev-wallet

```
Error: Objects are not valid as a React child
React child (found: object with keys {f_type, f_vsn, addr, cid, loggedIn, expiresAt, services})

The above error occurred in the <pre> component:
```

**Review by gregsantos — CHANGES_REQUESTED** [review]
merge latest so we can check the staging deploy

File: pages/index.js (hunk 2 above)
Comment (JeffreyDoyle) [reply pages/index.js:108]: Fixed!

File: pages/index.js [hunk 3]
```diff
@@ -79,6 +90,21 @@ export default function Home() {
           </span>
         ))}
       </div>
+      <div style={{ marginTop: "12px" }}>
+        <label htmlFor="manual-wallet">
+          Manually set "discovery.wallet" config:{" "}
```
Comment (gregsantos) [inline pages/index.js:95]:
```suggestion
          {`Manually set "discovery.wallet" config: `}
```

**Review by gregsantos — APPROVED** [review]
one lil linter thing, but looks good now

**Pull request merged**